homeassistant/components/zha/diagnostics.py

```diff
@@ -50,6 +50,15 @@
 CLUSTER_DETAILS = "cluster_details"
 UNSUPPORTED_ATTRIBUTES = "unsupported_attributes"
 
+BELLOWS_VERSION = version("bellows")
+ZIGPY_VERSION = version("zigpy")
+ZIGPY_DECONZ_VERSION = version("zigpy-deconz")
+ZIGPY_XBEE_VERSION = version("zigpy-xbee")
+ZIGPY_ZNP_VERSION = version("zigpy_znp")
```

Comment:
This one seems like the odd one out, we don't use the package name for it

```suggestion
ZIGPY_ZNP_VERSION = version("zigpy-znp")
```